[#1063] Fix docs for status command (#1064)
airyhq/airy: docs/docs/getting-started/deployment/vagrant.md

### Status

> Run the following `status command` for printing the information on how to access Airy Core.

Reviewer: ```suggestion
Run the following `status command` to print the information on how to access Airy Core.
```

"curl -X POST -H 'Content-Type: application/json' -d '{\"first_name\": \"Grace\",\"last_name\": \"Hopper\",\"password\": \"the_answer_is_42\",\"email\": \"grace@example.com\"}'
```

> For viewing the status of the Vagrant box run:

Reviewer: ```suggestion
To inspect the status of the Vagrant box run:
```
